content/en/docs/concepts/architecture/controller.md

```diff
@@ -92,9 +92,8 @@ Controllers that interact with external state find their desired state from
 the API server, then communicate directly with an external system to bring
 the current state closer in line.
 
-(There actually is a controller that horizontally scales the
-nodes in your cluster. See
-[Cluster autoscaling](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaling)).
+(There actually is a [controller](https://github.com/kubernetes/autoscaler/)
```

Comment:
nit: Do you need to enclose this sentence in parens?

Reply:
I want the reader to see that this is an aside or otherwise realize that if they aren't following this bit, they're OK to skip it.

Reply:
nit: See the page preview. The text seems out of place.
Could you add more text here or push this link to the "What's next" section?